demos/text_detection_demo/cpp/README.md

```diff
@@ -36,11 +38,20 @@ This file can be used as a parameter for [Model Downloader](../../../tools/downl
 * decoder_type = ctc
   * text-recognition-0012
   * text-recognition-0013
```

Comment:
```suggestion
```
model replaced with text-recognition-0014

Reply:
Should all mentions of `text-recognition-0014` be removed?

Reply:
Ah, found, it was closed